CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Fixed`
 
+- improved json schema as part of nf-core hackathon 2025
```

Comment:
Can you make this changelog entry look like the other entries?

Reply:
Do you mean to add a PR number?

Reply:
Yes :)